### ISSUE TITLE
[BUG] Invalid path

### ISSUE THREAD

**Issue by VishwaJay** [fantasycalendar/Fantasy-Calendar#289]
**Describe the bug**
```
error: invalid path 'resources/images/homepage/cederic-vandenberghe-21DP3hytVHw-unsplash.jpg:Zone.Identifier'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```

It reports that the clone succeeded, but it didn't. There is about 1/3 of the code missing after that point, since it truncates anything that wasn't checked out beyond that point in the stream. I don't have a functional copy to look at.

**To Reproduce**
1. Open the project in GitHub on Chrome, Firefox, Edge, etc., and note that it still happens on new installs.
2. Click the green "code" button.
3. Click "open with GitHub Desktop"
4. Wait for the download to complete. The error is consistent across all browsers.

**Expected behavior**
No error is thrown, and the resource clones/downloads correctly.

**Desktop (please complete the following information):**
Every major browser on Windows 10/11 seems to be throwing this error. It has been consistent for over a year, through updates, and in spite of moving to 2 new machines.

**Additional context**
As I code things for the web all the time (and commit to a few *nix-only projects on that basis), I know the likely cause is probably a simple typo, and easy enough to clear up.

This prevents my being able to look at the code unless I download it. I can't fork or contribute due to this error. I can't look at your code to tell if the bug is in my settings or not. And this means no forking directly on GitHub, even if I can manually create a fork. It prevents giving you credit on my fork (at least, where GitHub is concerned).

In short, it creates a lot of issues to have one file that throws an error due to resource invalidity.

**Comment by V13Axel** [comment]
Hey there,

Sorry about that, it's actually not a typo - It was a `Zone.identifier` file that represents an "alternate data stream", basically a feature in NTFS that rears its head as an actual file when someone downloads files into WSL2 file systems.

Thanks for pointing that out, you should be able to clone the repo now!
